model-api-gen-gradle-test/src/test/kotlin/org/modelix/metamodel/generator/GeneratedApiTest.kt

```diff
@@ -55,15 +60,28 @@ class GeneratedApiTest {
         val foundDeprecatedNodeChildLink = ClassConcept::class.members.any { it.hasDeprecationWithMessage() }
         val foundDeprecatedNodeReference = SuperInterfaceMethodCall_old::class.members.any { it.hasDeprecationWithMessage() }
 
-        assert(foundDeprecatedConcept)
-        assert(foundDeprecatedProperty)
-        assert(foundDeprecatedChildLink)
-        assert(foundDeprecatedReference)
+        assertTrue(foundDeprecatedConcept)
+        assertTrue(foundDeprecatedProperty)
+        assertTrue(foundDeprecatedChildLink)
+        assertTrue(foundDeprecatedReference)
 
-        assert(foundDeprecatedNodeWrapper)
-        assert(foundDeprecatedNodeProperty)
-        assert(foundDeprecatedNodeChildLink)
-        assert(foundDeprecatedNodeReference)
+        assertTrue(foundDeprecatedNodeWrapper)
+        assertTrue(foundDeprecatedNodeProperty)
+        assertTrue(foundDeprecatedNodeChildLink)
+        assertTrue(foundDeprecatedNodeReference)
+    }
+
+    @Test
+    fun `metaProperty alias is generated`() {
+        val hasAlias = IMetaConceptProperties::class.members.any { it.name == ConceptData.ALIAS_KEY }
+        assertTrue(hasAlias)
+        assertNull(C_BaseConcept.alias)
```

Comment:
Isn't the last line already sufficient for this test? Without the interface being generated, this tests wouldn't compile.

Reply:
It's true, it wouldn't compile in that case. But this also ensures that the alias property exists in the interface and isn't just directly defined in the concepts.